src/ImageSharp/Processing/Processors/Convolution/BoxBlurProcessor.cs

```diff
@@ -66,36 +66,7 @@ private DenseMatrix<float> CreateBoxKernel(bool horizontal)
                 ? new DenseMatrix<float>(size, 1)
                 : new DenseMatrix<float>(1, size);
 
-            float sum = 0F;
-            for (int i = 0; i < size; i++)
-            {
-                float x = 1;
-                sum += x;
-                if (horizontal)
-                {
-                    kernel[0, i] = x;
-                }
-                else
-                {
-                    kernel[i, 0] = x;
-                }
-            }
-
-            // Normalize kernel so that the sum of all weights equals 1
-            if (horizontal)
-            {
-                for (int i = 0; i < size; i++)
-                {
-                    kernel[0, i] = kernel[0, i] / sum;
-                }
-            }
-            else
-            {
-                for (int i = 0; i < size; i++)
-                {
-                    kernel[i, 0] = kernel[i, 0] / sum;
-                }
-            }
+            kernel.Fill(1.0F / size);
```

Comment:
Very silly of me eh? 😄

Reply:
nobody is perfect - and when changing existing code it's often hard to see when something get's simple enough to collapse in complexity.
Let's just call your version here "legacy code" and everything is forgiven ;)